arangod/Aql/Ast.h

```diff
@@ -509,12 +509,16 @@ class Ast {
   AstNode* optimizeObject(AstNode*);
 
 public:
+  /** Make sure to replace the AstNode* you pass into TraverseAndModify
+   *  if it was changes. This is necessary because the function itself
+   *  has only access to to the node but not its parent / owner.
+   */
```

Comment:
please doxygenify this properly.

Reply:
Also "if it was changes" -> "if it was changed"